test/AppStatsTest.php

```diff
@@ -31,6 +31,8 @@ protected function setUp() {
     }
 
     public function testPublishEventsForwards() {
+        echo '== testPublishEventsForwards()';
```

Comment:
This is everywhere in the specs. We probably need to clean it up, but that is consistent with the rest of the library specs.